style.css

```diff
@@ -31,15 +31,19 @@ hr {
 .card {
     background: var(--panel-bg-color);
     border: 1px var(--border-color) solid;
-    margin: 8px;
     padding: 4px 12px;
     position: relative;
     border-radius: 8px;
     max-width: 800px;
-    margin: 8px auto;
+    margin: 24px auto;
```

Comment:
What's the standard way to express spacing in a resolution independent manner? I would imagine pt or em?

Reply:
px is actually fine